src/main.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, width, height, title):
         super().__init__(
             width, height, title, enable_polling=True, update_rate=TICK_DURATION
         )
-
+        self.physics_engine = None
```

Comment:
Is this necessary?  It may have snuck in from some experiment or conversation.

Reply:
it was me, i was trying to equate what was needed from looking at the tutorials, but we have things set up a little different i think. I removed it when I realized we didn't need it.